src/EnergyPlus/WeatherManager.cc

```diff
@@ -237,7 +237,7 @@ namespace WeatherManager {
         Real64 constexpr beta = 0.000214;         // water thermal expansion coefficient, from engineeringtoolbox.com, 1/C
         Real64 constexpr assumedSurfaceTemp = 25; // Grashof requires a surface temp, this should suffice
         Real64 const localGrashofNumber =
-            (gravity * beta * (assumedSurfaceTemp - curWaterTemp) * pow(distanceFromBottomOfHull, 3)) / pow(waterKinematicViscosity, 2);
+            (gravity * beta * std::abs(assumedSurfaceTemp - curWaterTemp) * pow(distanceFromBottomOfHull, 3)) / pow(waterKinematicViscosity, 2);
```

Comment:
Tests are failing because they re called with a curWaterTemp of 30.0, so with this **abitrary** surface temp of 25 you end up with a (very) negative localGrashofNumber, and therefore problem where computing the pow(localGrashofNumber / 4, 0.25)

Reply:
OK.  Yeah, I believe the idea was that the water wouldn't be that warm, but the code should handle the situation properly.  Good stuff here.